### DISCUSSION TITLE
Is project still active?

### DISCUSSION THREAD

**Discussion by MagicAndre1981** [luminous-software/start-page-plus#50] [General]
Is project still active? Are you fit again to work on it?

**Comment by yannduran** [comment]
Yes, the project **is** still active. Funny that you ask the question now because I've just started working on it a couple of days ago to use the new [Visual Studio Community Toolkit](https://github.com/VsixCommunity/Community.VisualStudio.Toolkit) that @madskristensen and the VS extender community have been working on. 

I'll continue to work on _Start Page+_ as my health permits.

The next release will have the ability to edit a _recent item_'s path (the work's done), as well as your filtering request (I'll be working on that after the integration with the toolkit), hopefully a couple of other items as well. 

I've updated the [v1.1](https://github.com/luminous-software/start-page-plus/milestone/6) milestone to include what's coming soon, and pushed some items to [v1.2](https://github.com/luminous-software/start-page-plus/milestone/7) and [v1.3](https://github.com/luminous-software/start-page-plus/milestone/9).

Thanks for the enquiry!

**Reply by MagicAndre1981** [reply]
ok, I would love to see automatic close/open on solution open/close and disabled developer news. Everything else can be in 1.2

**Reply by MagicAndre1981** [reply]
any update? can you please push your changes so that we can work on VS2022 version. currently  VS2022 is unusable without this extension
